src/resources/postcss/base/forms/_tabs.pcss

```diff
@@ -68,18 +69,36 @@
 			&:first-child label {
 				@mixin form-control-select-list-item-first;
 
+				body#top & { /* Enfold compatibility */
+					@mixin form-control-select-list-item-first;
```

Comment:
Are all the mixin parameters overridden by enfold, or some of them? If we can help it, we should only target those affected.

Reply:
Yes, those mixins are only paddings. I can set the properties there directly in case the mixins grow in the future.

Reply:
ah okay, if they are all affected then no worries, just wanted to check.